operator/pkg/util/util.go

```diff
@@ -56,29 +56,31 @@ func GetNamespaceObjBytes(clientNs string) ([]byte, error) {
 }
 
 func FilterExistingResources(resources kube.ResourceList) (kube.ResourceList, error) {
-	var existingResources kube.ResourceList
-
-	err := resources.Visit(func(info *resource.Info, err error) error {
-		if err != nil {
-			return err
-		}
-
+	existingResources := kube.ResourceList{}
```

Comment:
was there a reason to not use `resources.Visit` or just for simplicity?

Reply:
Problem with resources.Visit is that it contains an err interface and I would then have to still collect an error array and always return nil in the Visit inline function. That looked really, really ugly, so I just swapped it to a for loop. If its important, then I can switch it back.